src/styles/blocks/calendar-loops.scss

```
@@ -0,0 +1,23 @@
+@each $day, $margin in $days {
+  .calendar--#{$day} > .calendar__day:first-child {

```

This selector can be simplified

```suggestion
  .calendar--#{$day} :first-child {

```

---

In this case first-child will be used to calendar block instead of calendar__day.